yarn.lock

```diff
@@ -1627,20 +1695,20 @@
   integrity sha512-OGDAWbAcm4mh+wzw6uBYxw0OJtqJ+39W3pOXl0W93d2p/cH7ppfvunwf7irEPw0FPiAIuR/F77nU1zMKNhm8Dg==
 
 "@parity/light.js-react@^5.1.1":
-  version "5.1.1"
-  resolved "https://registry.yarnpkg.com/@parity/light.js-react/-/light.js-react-5.1.1.tgz#1b5e1b62dd04fa846e8d11971742ec793500fe0c"
-  integrity sha512-LD05kpHLzhgONeFiGEKrN59LBSmeBzx5NakxBXzu8vixBQt6Cyq5z1gOzg4RtbtkcTrUC7Q02H4DwOh8QJpNmw==
+  version "5.1.3"
+  resolved "https://registry.yarnpkg.com/@parity/light.js-react/-/light.js-react-5.1.3.tgz#f7e28a0ecbd05246c99cc3f20b4c8cf96c44210f"
+  integrity sha512-t7BTy7ECrPJzoXle3gOWurK47+PEltHrfajJStvFgzVG0oyKKyrvb5ro3w2YIOc51L9tfHX0eeS1tuvUeCJLvQ==
   dependencies:
     recompose "^0.30.0"
     symbol-observable "^1.2.0"
 
 "@parity/light.js@^5.1.1":
-  version "5.1.1"
-  resolved "https://registry.yarnpkg.com/@parity/light.js/-/light.js-5.1.1.tgz#141f4c168f46208b1caa6273161ad9a8b1675862"
-  integrity sha512-AJ/klFpim+lKLxYKpIhJL9Btu4OuXs43lfbBK8XA0kXOV6M51vCv5n+sO5oTWVx6IJE3lXuDIc674H9BC4fcvQ==
+  version "5.1.3"
```

Comment:
AH I didn't see this, I merged too fast... `@parity/*` should NOT have been updated, with this update we cannot send a tx. I'll add a patch asap